receiptui/src/test/java/com/hyperwallet/android/ui/receipt/viewmodel/ListUserReceiptsViewModelTest.java

```diff
@@ -0,0 +1,80 @@
+package com.hyperwallet.android.ui.receipt.viewmodel;
+
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+
+import com.hyperwallet.android.ui.receipt.repository.PrepaidCardReceiptRepository;
+import com.hyperwallet.android.ui.receipt.repository.PrepaidCardReceiptRepositoryImpl;
+import com.hyperwallet.android.ui.receipt.repository.UserReceiptRepository;
+import com.hyperwallet.android.ui.receipt.repository.UserReceiptRepositoryImpl;
+
+import org.hamcrest.CoreMatchers;
+import org.hamcrest.MatcherAssert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+
+@RunWith(RobolectricTestRunner.class)
+public class ListUserReceiptsViewModelTest {
+
+    private ListUserReceiptsViewModel mListUserReceiptsViewModel;
+    private PrepaidCardReceiptRepository mPrepaidCardReceiptRepository;
+    private UserReceiptRepository mUserReceiptRepository;
+    private ListReceiptsViewModel.ListReceiptsViewModelFactory mListReceiptsViewModelFactory;
+
+    @Before
+    public void initializedViewModel() {
+        String prepaidToken = "trm-ppc-token";
+        mPrepaidCardReceiptRepository = spy(new PrepaidCardReceiptRepositoryImpl(prepaidToken));
+        mUserReceiptRepository = spy(new UserReceiptRepositoryImpl());
+        mListReceiptsViewModelFactory = new ListReceiptsViewModel.ListReceiptsViewModelFactory(prepaidToken,
+                mUserReceiptRepository, mPrepaidCardReceiptRepository);
+        mListUserReceiptsViewModel = new ListUserReceiptsViewModel(mUserReceiptRepository);
+    }
+
+    @Test
+    public void testIsLoadingData_returnsLiveData() {
+        assertThat(mListUserReceiptsViewModel.isLoading(), is(notNullValue()));
+    }
+
+    @Test
+    public void testGetReceiptList_returnsLiveData() {
+        assertThat(mListUserReceiptsViewModel.receipts(), is(notNullValue()));
+    }
+
+    @Test
+    public void testGetDetailNavigation_returnsLiveData() {
+        MatcherAssert.assertThat(mListUserReceiptsViewModel.getDetailNavigation(),
+                CoreMatchers.is(CoreMatchers.notNullValue()));
```

Comment:
use static imports, Don't use `MatcherAssert.assertThat` or `CoreMatchers.is` or `CoreMatchers.notNullValue()`
it should be simple as `assertThat(` or `is(` or `notNullValue(`
Revisit all your test code and fix wherever applicable